src/graphx/graphx.h

```diff
@@ -982,8 +994,9 @@ void gfx_PrintChar(const char c);
 /**
  * Prints a signed integer.
  *
- * Outputs at the current cursor position. Padded with leading zeros if
+ * Outputs at the current cursor position. Pads the integer with leading zeros if
  * necessary to satisfy the specified minimum length.
+ * For example, printInt(5,3) prints "005".
```

Comment:
I think you mean "gfx_PrintUInt" here

Reply:
Yeah, I changed it to gfx_PrintInt (it's in)